addons/mail/wizard/mail_compose_message_view.xml

```diff
@@ -75,6 +75,7 @@
                     </group>
                     <footer>
                         <button string="Send" attrs="{'invisible': [('is_log', '=', True)]}" name="send_mail_action" type="object" class="btn-primary o_mail_send"/>
+                        <button string="Send and schedule next" invisible="not context.get('from_activity', False)" name="send_mail_and_schedule_next_action" type="object" class="btn-primary o_mail_send"/>
```

Comment:
We try to name space context keys (at least I do it). Maybe we could have mail_compose_from_activity ? I know it is longer but it is "mail" namespaced :) .